frontend/src/pages/Landing/RegionalSelect.js

```diff
@@ -0,0 +1,156 @@
+
+const styles = {
+  container: (provided, state) => {
+    // To match the focus indicator provided by uswds
+    const outline = state.isFocused ? '0.25rem solid #2491ff;' : '';
+    return {
+      ...provided,
+      outline,
+    };
+  },
+  input: () => ({ display: 'none' }),
+  control: (provided) => ({
+    ...provided,
+    borderColor: '#0166AB',
+    backgroundColor: '#0166AB',
+    borderRadius: '5px',
+    paddingLeft: '5px',
+    paddingTop: '4px',
+    paddingBottom: '4px',
+    whiteSpace: 'nowrap',
+    color: 'white',
+    width: '120px',
+  }),
+  indicatorSeparator: () => ({ display: 'none' }),
+  menu: (provided) => ({
+    ...provided,
+    width: '200px',
+  }),
+  // placeholder: () => ({
+  //   color: 'white', fontWeight: 600, fontSize: '17px', marginRight: '-5px',
+  // }),
+  option: (provided, state) => ({
+    ...provided,
+    color: state.isSelected ? '#0166AB' : 'black',
+    fontWeight: state.isSelected ? '700' : 'normal',
+    backgroundColor: state.isSelected ? '#F8F8F8' : '#FFFFFF',
+    padding: 11,
+  }),
+  singleValue: (provided) => {
+    const single = { color: '#FFFFFF', fontWeight: 600 };
+
+    return {
+      ...provided, ...single,
+    };
+  },
+  valueContainer: () => ({ padding: '10px 8px' }),
+};
+
+function RegionalSelect(props) {
+  const {
+    regions, onApply,
+  } = props;
+
+  const [selectedItem, setSelectedItem] = useState();
+  const [appliedItem, setAppliedItem] = useState();
+  const [menuIsOpen, setMenuIsOpen] = useState(false);
+
+  // const delayedCloseMenu = () => setTimeout(setMenuIsOpen(false), 1000);
+
+  const CustomOption = (props1) => {
+    const {
+      data, innerRef, innerProps, isSelected,
+    } = props1;
+    return data.custom ? (
+      <div ref={innerRef} {...innerProps}>
+        <Button
+          type="button"
+          className="float-left margin-2 smart-hub--filter-button"
+          onClick={() => {
+            onApply(selectedItem);
+            setAppliedItem(selectedItem);
+            setMenuIsOpen(false);
+          }}
+        >
+          Apply
+        </Button>
+      </div>
+    ) : (
+      <components.Option {...props1}>
+        {data.label}
+        {isSelected && (
+          <img
+            className="tta-smarthub--check"
+            src={check}
+            style={{
+              width: 32,
+              float: 'right',
+              marginTop: '-9px ',
+            }}
+            alt={data.label}
+          />
+        )}
+      </components.Option>
+    );
+  };
+
+  CustomOption.propTypes = {
+    // eslint-disable-next-line react/forbid-prop-types
+    data: PropTypes.object.isRequired,
+    innerRef: PropTypes.string.isRequired,
+    // eslint-disable-next-line react/forbid-prop-types
+    innerProps: PropTypes.object.isRequired,
+  };
+
+  const options = [...getUserOptions(regions), { custom: true }];
+  return (
+    <Select
```

Comment:
Sorry to jump back in to flag another issue, but I am using this in what I'm working on and having trouble keyboard focusing this select box. I don't immediately see a prop you can pass to react-select to easily fix this, but I've only briefly looked and will continue to do so.

Reply:
This wasn't my top priority for this PR mainly because of the limited audience. How about we address this separately, so that this PR goes in? That way they will at least have access to the data. Thanks for flagging it though.